pkgs/applications/terminal-emulators/alacritty/default.nix

```diff
@@ -129,11 +129,12 @@ rustPlatform.buildRustPackage rec {
   passthru.tests.test = nixosTests.terminal-emulators.alacritty;
 
   meta = with lib; {
+    changelog = "https://github.com/alacritty/alacritty/blob/v${version}/CHANGELOG.md";
```

Comment:
Moving this line isn't really relevant to the change specified. Would you mind reverting it to clean up the diff?